src/main/java/gregtech/api/util/GTControlledRegistry.java

```diff
@@ -57,9 +57,6 @@ public void register(int id, K key, V value) {
         if (id < 0 || id >= maxId) {
             throw new IndexOutOfBoundsException("Id is out of range: " + id);
         }
-        if (key instanceof ResourceLocation) {
-            key = (K) GameData.checkPrefix(key.toString(), false);
-        }
```

Comment:
Reason for this deletion?

Reply:
allow other mod to register their things in their domain

Reply:
Ah yeah, instead of doing this, we need to implement `IContextSetter` on `RegisterEvent`. Then the active mod container will be automatically set.

Reply:
if we use the MBT to register things with ZS scripts. they all will be registered to the `crafttweaker:xxxxx`

Reply:
It may not be the best approach, but it definitely doesn't introduce problems